src/pages/main/components/selected-dataset/components/dataset-info/dataset-info.utils.tsx

```diff
@@ -0,0 +1,88 @@
+import styles from './dataset-info.module.css'
```

Comment:
Utils are utility functions, these are React components. Please split them onto separate files and move them to /components folder.

Reply:
Fixed